docs/source/FrevaCli.rst

```diff
@@ -512,6 +512,63 @@ of the last 3 plugin applications:
       for file in hist["result"].keys():
          print(file)
 
+
+By default, Freva only shows your user's history. However,
+with the ``--all-users`` flag, the query is extended to all users.
+For example, while the following Freva command shows the last job run
+by you:
+
+.. code:: console
+
+    freva history --limit 1 --json
+
+
+.. execute_code::
```

Comment:
here `DummyUser` seems to be redundant. And also to show all users' history, what is ur opinion to remove `--limit 1` from `freva history --limit 1 --json --all-users` and on the other hand to show the ability of `--all-users` flag make a list of diff users with a list of objects?

```rst

.. execute_code::
    :hide_code:

    import freva
    import random
    import string
    import datetime
    import socket
    from django.contrib.auth.models import User
    from evaluation_system.model.history.models import History
    from subprocess import run, PIPE
    freva.run_plugin("dummypluginfolders")
    for i in range(2):
        random_string = "".join(random.choices(string.ascii_lowercase + string.digits, k=10))
        user = User.objects.create_user(
            username=f"user_{random_string}", password="123"
        )
        History.objects.create(
            timestamp=datetime.datetime.now(),
            status=History.processStatus.running,
            uid=user,
            configuration='{"some": "config", "dict": "values"}',
            tool=f"dummytool_{random_string}",
            slurm_output="/path/to/slurm-44742.out",
            host=socket.gethostbyname(socket.gethostname()),
        )
        user.delete()
    res_my_user = run(["freva", "history", "--limit", "1", "--json"], check=True, stdout=PIPE, stderr=PIPE)
    print(res_my_user.stdout.decode())


The following one will also take in account other users:

.. code:: console

    freva history --limit 1 --json --all-users

.. execute_code::
    :hide_code:

    from subprocess import run, PIPE
    res_all_users = run(["freva", "history", "--json", "--all-users"], check=True, stdout=PIPE, stderr=PIPE)
    print(res_all_users.stdout.decode())
```

Reply:
I am confused. Isn't this PR about gifs?

Reply:
yes, I'm with you. I think we need to change the title and description to make it clear and on the other hand document the development in a clear way